### ISSUE TITLE
2 Modes and Fan Speed are wrong

### ISSUE THREAD

**Issue by Homey4Osty** [ugumba/homey-panasonic-comfort-cloud-alt#18]
Hello. It's me again, Osty. (Sorry ;-) )

The new version (1.1.6) works fine. Thanks a lot for it!

Today I added all my 4 room-air-conditioners to homey. And I'm able to start/stop them or change the temperature.
But: I can set the Operations-Mode, the Fan-Speed and the Eco-Mode only for **one (and it's always the same) air conditioner**.

![image](https://github.com/ugumba/homey-panasonic-comfort-cloud-alt/assets/146454771/8186bb34-fbdc-49f1-9e12-bc5cef030936)

The 'Then-Cards' for the modes and fan speed are not for the room-devices: 
![image](https://github.com/ugumba/homey-panasonic-comfort-cloud-alt/assets/146454771/76dd5171-e877-4831-a5ee-62c5c80a452f)

Are you able to add this "then-cards" to the device / air conditioner?

Thanks again!

**Comment by ugumba** [comment]
Thanks for reporting!

Another user is apparently [preparing](https://community.homey.app/t/app-pro-panasonic-comfort-cloud-alternative/75906/137?u=robert_schmidt) a pull request to solve this issue!

**Comment by jarll** [comment]
Yes, I have a working fix (on the old code base) but I'm on vacation and thus unable to test it with latest authentication fixes (although the should be unrelated) and push it to my forked repo until July 20th.

**Comment by jarll** [comment]
Pull request from https://github.com/jarll/homey-panasonic-comfort-cloud-alt/tree/adddeviceactioncards created
